inc/patterns/about-me-logo-header-description-and-social-on-left-menu-and-separator-on-right.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * About Me - Logo, header, description, and social on left, menu and separator on right
+ */
+return array(
+	'title'      => __( 'About Me - Logo, header, description, and social on left, menu and separator on right', 'twentytwentytwo' ),
+	'categories' => array( 'twentytwentytwo-about-me' ),
+	'content'    => '<!-- wp:cover {"overlayColor":"foreground","minHeight":100,"minHeightUnit":"vh","contentPosition":"center center","align":"full","style":{"spacing":{"padding":{"top":"4rem","right":"4rem","bottom":"4rem","left":"4rem"}}}} -->
+					<div class="wp-block-cover alignfull has-foreground-background-color has-background-dim" style="padding-top:4rem;padding-right:4rem;padding-bottom:4rem;padding-left:4rem;min-height:100vh"><div class="wp-block-cover__inner-container"><!-- wp:navigation {"itemsJustification":"right"} -->
+					<!-- wp:page-list {"isNavigationChild":true,"showSubmenuIcon":true,"openSubmenusOnClick":false} /-->
+					<!-- /wp:navigation -->
+
+					<!-- wp:columns -->
+					<div class="wp-block-columns"><!-- wp:column {"verticalAlignment":"bottom","width":"45%","style":{"spacing":{"padding":{"top":"12rem","right":"2rem","bottom":"2rem","left":"2rem"}}}} -->
+					<div class="wp-block-column is-vertically-aligned-bottom" style="padding-top:12rem;padding-right:2rem;padding-bottom:2rem;padding-left:2rem;flex-basis:45%"><!-- wp:site-logo {"width":60} /-->
+
+					<!-- wp:heading {"style":{"typography":{"fontWeight":"300","lineHeight":"1.115","fontSize":"72px"}}} -->
+					<h2 style="font-size:72px;font-weight:300;line-height:1.115"><em>' . __( 'Jesús<br>Rodriguez', 'twentytwentytwo' ) . '</em></h2>
```

Comment:
this is not escaped, we can do it like so:
```suggestion
					<h2 style="font-size:72px;font-weight:300;line-height:1.115"><em>' . wp_kses_post(  __( 'Jesús<br>Rodriguez', 'twentytwentytwo' ) ) . '</em></h2>
```